cmd/aws-iam-authenticator/server.go

```diff
@@ -52,15 +53,16 @@ var serverCmd = &cobra.Command{
 		var err error
 		fmt.Printf("Authenticator Version: %q, %q\n", pkg.Version, pkg.CommitID)
 		metrics.InitMetrics(prometheus.DefaultRegisterer)
-		stopCh := signals.SetupSignalHandler()
+		cmd.Context()
```

Comment:
@micahhausler what are we doing with this?  https://github.com/spf13/cobra/blob/main/command.go#L257-L266

Reply:
I think we can drop it